tools/react-app-template/src/backend.ts

```diff
@@ -1,9 +1,9 @@
 // (C) 2022 GoodData Corporation
 import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
 import { withCaching, RecommendedCachingConfiguration } from "@gooddata/sdk-backend-base";
-import bearFactory, { AnonymousAuthProvider } from "@gooddata/sdk-backend-bear";
+import backendFactory, { AnonymousAuthProvider } from "@gooddata/sdk-backend-tiger";
 
 export const backend: IAnalyticalBackend = withCaching(
-    bearFactory().withAuthentication(new AnonymousAuthProvider()),
+    backendFactory().withAuthentication(new AnonymousAuthProvider()),
```

Comment:
@kandl Do we really need anonymous auth provider? Wouldn't context deferred one just work? Given we create a separate user account on Tiger and use it to generate Token (so that user details return something generic and not one of our names).
I'm asking because having proper auth provider from the start will mean one less step in the guide on how to switch to own data instead of demo data.